src/ol/renderer/layerrenderer.js

```diff
@@ -151,7 +151,7 @@ ol.renderer.Layer.prototype.loadImage = function(image) {
     // listener (a noop if the listener was already registered)
     goog.asserts.assert(imageState == ol.ImageState.IDLE ||
         imageState == ol.ImageState.LOADING);
-    goog.events.listenOnce(image, goog.events.EventType.CHANGE,
+    goog.events.listen(image, goog.events.EventType.CHANGE,
```

Comment:
> Ah good point, so yes, this is an issue indeed

I am not sure :-)

So, with your changes, "change" listeners are not removed from `ol.Image` objects. But this may not be a problem, as it may not prevent the GC from garbage-collecting `ol.Image` objects that aren't used anymore. I'd suggest verifying this before merging this.